_includes/aiml-sidebar.html

```diff
@@ -1,74 +1,65 @@
 <div id="desktop-sidebar">
-    {% for lesson in site.aiml %}
-        {% assign pNum = page.num | plus: 0 %}
-        {% assign x = lesson.num | plus: 0 %}
-        {% if x < pNum %} 
-            <div class="sidebar-item">
-            <a class="ai-link" href="{{lesson.url}}">#{{lesson.num}} {{lesson.title}}</a>
-            </div>
-        {% elsif x == pNum and lesson.num == page.num %}
-            <div class="sidebar-item">
-            <span class="current-dot"></span>
-            <a class="ai-link" href="javascript:void(0)">#{{lesson.num}} {{lesson.title}}</a>
-            </div>
-        {% else %}
-            <div class="sidebar-item">
-            <a class="ai-link" href="{{lesson.url}}">#{{lesson.num}} {{lesson.title}}</a>
-            </div>
-       {% endif %}
-    {% endfor %}
+	{% for lesson in site.aiml %}
+	<div class="sidebar-item">
+		{% assign lessonUrl = lesson.url%}
+		{% if lesson.num == page.num %}
+		{% assign lessonUrl = "javascript:void(0)" %}
+		<span class="current-dot"></span>
+		{% endif %}
+
+		<a class="ai-link" href="{{lessonUrl}}">#{{lesson.num}} {{lesson.title}}</a>
+	</div>
+	{% endfor %}
 </div>
 
 <div id="mobile-sidebar">
-    <div id="sidebar-btn-container">
-        <div id="open-mobile-icon" class="mobile-toggle-btn">Lessons</div>
-        <div id="close-mobile-icon" class="mobile-toggle-btn">Close</div>
-    </div>
-    <div id="hamburger-content">
-        {% for lesson in site.aiml %}
-            {% assign pNum = page.num | plus: 0 %}
-            {% assign x = lesson.num | plus: 0 %}
-            {% if x < pNum %} 
-                <div class="hamburger-item">
-                    <a class="ai-link" href="{{lesson.url}}">#{{lesson.num}} {{lesson.title}}</a>
-                </div>
-            {% else if x > pNum %}
-                <div class="hamburger-item">
-                    <a class="ai-link" href="{{lesson.url}}">#{{lesson.num}} {{lesson.title}}</a>
-                </div>
-            {% else %}
-                <div class="hamburger-item">
-                    <a class="ai-link" href="javascript:void(0)">#{{lesson.num}} {{lesson.title}}</a>
-                </div>
-            {% endif %}
-        {% endfor %}
-    </div>
+	<div id="sidebar-btn-container">
+		<button id="open-mobile-icon" onclick="toggleHamburgerContent()" class="d-flex">Lessons</button>

```

Comment:
Super quick note - can we add the `onclick` using JS (e.g. with `element.onclick = ...`? I prefer to put all of our JS related things in just pure JS; it can help make things less confusing!

(same for the other button too!)

Reply:
Ok, I think I did that. Is this the correct way to do it?
`document.getElementById("open-mobile-icon").onclick = toggleHamburgerContent;`